pkg/agent/agent_linux.go

```diff
@@ -43,6 +43,8 @@ func (i *Initializer) prepareHostNetwork() error {
 	return nil
 }
 
+const maxMTUSupportedBySuricata = 32678
```

Comment:
Better to have how a link of the source code how it's calcuated.

Reply:
Added.